pkg/ccl/logictestccl/testdata/logic_test/multi_region_backup

```diff
@@ -957,3 +957,28 @@ TABLE regional_by_table_in_ca_central_1  ALTER TABLE regional_by_table_in_ca_cen
                                          constraints = '{+region=ap-southeast-2: 1, +region=ca-central-1: 1, +region=us-east-1: 1}',
                                          voter_constraints = '[+region=ca-central-1]',
                                          lease_preferences = '[[+region=ca-central-1]]'
+
+statement error cannot backup individual tables from a multi-region database
+BACKUP TABLE regional_by_row_table TO 'nodelocal://self/rbr_table/'
```

Comment:
I missed this in the last review, but I have a preference towards making these backup to `nodelocal://1/` instead in case we expand these tests to run on a multi-node cluster with different external storage directories.

We don't need to update these in this PR -- can be cleaned up in this file in a follow up.